examples/pipes.rb

```diff
@@ -33,7 +33,7 @@
 
 client.authenticate
 client.tree_connect("\\\\#{address}\\IPC$")
-pipe = client.create_pipe(pipename, nil)
+pipe = client.create_pipe(pipename)
```

Comment:
Since #create_pipe is only used in this example, I think we could just update it instead of modifying the actual method return value, which would require some changes on msf side. Maybe something like this would be enough:
```ruby
client.create_pipe(pipename)
pipe =  client.last_file
```

Reply:
I updated the return value as you mentioned.